### PR TITLE
workflows/pr_enclave_tls: update samples install path

### PR DESCRIPTION
**Pull request by haosanzi**
Signed-off-by: Shirong Hao <shirong@linux.alibaba.com>

### CODE REVIEW

File: .github/workflows/pr_enclave_tls.yml [hunk 1]
```diff
@@ -127,8 +129,8 @@ jobs:
     - name: Test enclave tls server for mutual nullquote
       run: |
         echo random-port3 ${{ steps.random-port-generator3.outputs.random-port }}
-        docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}' &
-        docker exec $inclavare_test bash -c 'sleep 10 && cd /opt/enclave-tls/bin/ && ./enclave-tls-client -l debug -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}'
+        docker exec $inclavare_test bash -c 'cd /usr/share/enclave-tls/samples && ./enclave-tls-server -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}' &
```
Comment (jiazhang0) [inline .github/workflows/pr_enclave_tls.yml:132]: Use `/usr/share/enclave-tls/samples/enclave-tls-server` directly.

Comment (haosanzi) [reply .github/workflows/pr_enclave_tls.yml:132]: Type `/usr/share/enclave-tls/samples/enclave-tls-server` does not work.
Unless add the command `cp /usr/share/enclave-tls/samples/sgx_stub_enclave.signed.so ./`

```
# /usr/share/enclave-tls/samples/enclave-tls-server
    - Welcome to Enclave-TLS sample server program
[ERROR] Failed to load enclave 8207
[ERROR] Failed to load sgx stub enclave
```

File: .github/workflows/pr_enclave_tls.yml [hunk 2]
```diff
@@ -94,8 +93,13 @@ jobs:
...
           libsgx-ra-network libsgx-ra-uefi libsgx-uae-service libsgx-urts sgx-ra-service \
-          sgx-aesm-service;
-        mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt'
+          sgx-aesm-service'
+
+    - name: install enclave-tls
+      run: docker exec $inclavare_test bash -c 'mv /root/inclavare-containers/${{ matrix.tag }}/enclave-tls /opt;
+        mkdir -p /usr/share/enclave-tls;
+        mv /root/inclavare-containers/${{ matrix.tag }}/samples /usr/share/enclave-tls;
+        cp /usr/share/enclave-tls/samples/sgx_stub_enclave.signed.so ./'
```
Comment (hustliyilin) [inline .github/workflows/pr_enclave_tls.yml:102]: This pr mainly solves the path of enclave-tls changing problem, So It's better to change 
- `/usr/share/enclave-tls/samples`  to a ENV like `ENCLAVE_TLS_BINDIR`
       -  so `/usr/share/enclave-tls` can be written like `$(dirname %{ENCLAVE_TLS_BINDIR})`
- `/opt/enclave-tls` to a ENV `ENCLAVE_TLS_ROOTDIR`

**Review by hustliyilin — APPROVED** [review]
LGTM

**Pull request merged**